react/features/base/translation/Translation.js

```diff
@@ -0,0 +1,46 @@
+/* global interfaceConfig */
+import i18n from 'i18next';
+import XHR from 'i18next-xhr-backend';
+import { DEFAULT_LANG, languages } from './constants';
+import languagesR from '../../../../lang/languages.json';
+import mainR from '../../../../lang/main.json';
+
+import LanguageDetector from './LanguageDetector';
+
+/**
+ * Default options to initialize i18next.
+ *
+ * @enum {string}
+ */
+const defaultOptions = {
+    compatibilityAPI: 'v1',
+    compatibilityJSON: 'v1',
+    fallbackLng: DEFAULT_LANG,
+    load: 'unspecific',
+    resGetPath: 'lang/__ns__-__lng__.json',
+    ns: {
+        namespaces: [ 'main', 'languages' ],
+        defaultNs: 'main'
+    },
+    lngWhitelist: languages.getLanguages(),
+    fallbackOnNull: true,
+    fallbackOnEmpty: true,
+    useDataAttrOptions: true,
+    app: typeof interfaceConfig === 'undefined'
+        ? 'Jitsi Meet' : interfaceConfig.APP_NAME
```

Comment:
do we support null or '' as the app name? How about `interfaceConfig.APP_NAME || 'Jitsi Meet'`